src/groovy/org/pih/warehouse/order/OrderStatus.groovy

```diff
@@ -32,6 +32,10 @@ enum OrderStatus {
         [PENDING, PLACED, PARTIALLY_RECEIVED, RECEIVED, COMPLETED, CANCELED, REJECTED]
     }
 
+    static displayStatusList() {
+        ['PENDING','PLACED','SHIPPED','PARTIALLY_RECEIVED','RECEIVED','CANCELED']
```

Comment:
It should not be strings.

Reply:
Wait a second. There is no SHIPPED status...

Reply:
You should concatenate displayStatusList form OrderStatus and displayStatusList ShipmentStatusCode

Reply:
@jmiranda What do you think about it? Should we concatenate displayStatusList from OrderStatus and ShipmentStatusCode or leave it as it as a list of strings in OrderStatus?

Reply:
I don't know. For now let's just use a list of strings, I guess.

Reply:
I agree with @awalkowiak that it would be a better solution but I'm not sure if all statuses on both objects apply to orders. So we'd have a superset of statuses which wouldn't be bad but some wouldn't apply. Also, I think the statuses depend on the order type (purchase order statuses are different from transfer order statuses).